Application/EdFi.Ods.Features/OpenApiMetadata/Factories/OpenApiMetadataParametersFactory.cs

```diff
@@ -115,13 +114,52 @@ public IDictionary<string, Parameter> Create(bool isCompositeContext)
                     {
                         name = "totalCount",
                         description =
-                            "Indicates if the total number of items available should be returned in the 'Total-Count' header of the response.  If set to false, 'Total-Count' header will not be provided.",
+                            "Indicates if the total number of items available should be returned in the 'Total-Count' header of the response.  If set to false, 'Total-Count' header will not be provided. Must be false when using key set paging.",
                         @in = "query",
                         type = "boolean",
                         required = false,
                         @default = false
                     }
                 );
+
+                parameters.Add(
+                    "pageToken", new Parameter
+                    {
+                        name = "pageToken",
+                        description = "Applicable when using key set paging. The token of the page to retrieve.",
```

Comment:
This text appears here:

![image](https://github.com/user-attachments/assets/723b1160-9baf-4baa-87d4-985db9f8d954)

Reply:
This change looks good

Reply:
Suggestion for this text: "The token of the page to retrieve, obtained either from the "Next-Page-Token" header of the previous request, or from the "partitions" endpoint for the resource."

Reply:
Changed.